src/pages/ListPage/ChartSection.jsx

```diff
@@ -2,8 +2,8 @@ import { useState } from "react";
 // Component
 import VoteButton from "components/Button/Button";
 import BorderButton from "components/Button/BorderButton";
```

Comment:
Button을 하나만 import하는 것은 어떨까요?
이렇게 사용할 수 있겠네요!

```suggestion
import Button from "components/Button";

// ...


<Button>차트 투표하기</Button>
<Button.Border className={styles.borderButton}>더 보기</Button.Border>
```